hcshell/console.cpp

```diff
@@ -140,17 +142,20 @@ void Console::run() {
     }
 
     if(m_interactive) {
-        std::cout << "> ";
-        std::string input;
-        std::getline(std::cin, input);
-        if(input == "exit")
+        char* buf = readline(">> ");
+        if(buf == nullptr || buf == "quit" || buf == "exit")
```

Comment:
This will most likely not work as that's not how to compare char* in C/C++

Reply:
confirmed that it doesn't work on linux